Odysee.xcodeproj/project.pbxproj

```diff
@@ -980,6 +985,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.odysee.Odysee;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_TREAT_WARNINGS_AS_ERRORS = YES;
```

Comment:
There was only one existing warning (related to `keyWindow` being deprecated in iOS 13), but in my experience it's much safer and leads to better code to have this turned on.